src/main/resources/buildServerResources/admin/buildScanAdminPage.jsp

```diff
@@ -0,0 +1,23 @@
+<%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+    <%@include file="/include-internal.jsp" %>
+    <jsp:useBean id="count" type="java.lang.Integer" scope="request"/>
+    <h3>Cleanup storage of previous plugin versions</h3>
```

Comment:
Should be same style/size as on the plugins list page

Reply:
Wouldn't it bet better to rely in the cleanup setting page?